style/js/shield_defs.js

```diff
@@ -2279,27 +2280,44 @@ export function loadShields(shieldImages) {
     2,
     1
   );
-
   [
-    "Amsterdam",
-    "Den Haag",
-    "Rotterdam",
-    "Nijmegen",
-    "Parkstad",
-    "Zaanstad",
+    "NL:S:Amsterdam",
+    "NL:S:Den Haag",
+    "NL:S:Nijmegen",
+    "NL:S:Parkstad",
+    "NL:S:Rotterdam",
+    "NL:S:Zaanstad",
+    "NL:binnenstedelijke_ring", // also for Curacao
```

Comment:
I would suggest extracting the NL city shield to a separate variable, extracting the `NL:S` prefixes in the same manner as noted for `NL:R`, and then just making a separate line for Curacao.  Just makes things a touch more organized (see examples in US states with county shields for other examples of this).

Reply:
Did this in the new commit (and clarified the comment regarding Curacao).